### ISSUE TITLE
dashborad port(8080) can not be created

### ISSUE THREAD

**Issue by ywyt738** [containous/traefik#5347]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

I'm using rc3 instead of rc2. I enable api but it can't listen 8080, so I can visit the dashboard.

my config file:
```
[api]
  dashboard = true
```

### What did you expect to see?

It can automatically create entryPoint like version RC2.

### What did you see instead?

In RC3, traefik don't listen 8080 for api in default.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
v2.0.0-rc3
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
[global]
  checkNewVersion = true
  sendAnonymousUsage = false

[entryPoints]
  [entryPoints.http]
    address = ":80"

  [entryPoints.https]
    address = ":443"

  [entryPoints.oldhttps]
    address = ":8443"

[log]
  level = "INFO"
  filePath = "log/traefik.log"

[accessLog]
  filePath = "/var/log/traefik.access.log"

[api]
  dashboard = true

[providers]
  [providers.file]
    filename = "dynamic.config.toml"
    watch = true
```

**Comment by ldez** [comment]
Hello,

We introduce a new security system for the API:
- https://community.containo.us/t/traefik-release-v2-0-0-rc3/1542
- https://github.com/containous/traefik/pull/5311
- https://docs.traefik.io/v2.0/operations/api/#configuration

**Comment by denysvitali** [comment]
TL;DR: 
```
[http.routers.my-api]
    rule="PathPrefix(`/api`) || PathPrefix(`/dashboard`)"
    service="api@internal"
    middlewares=["auth"]
```

Thanks @ldez for linking the specific documentation part 🚀
